### PR TITLE
change v1alpha1 from capi controlplane and minor changes

### PR DESCRIPTION
**Pull request by alknopfler**
- Change to move from v1alpha1 to v1beta1 in rke2controlplane 
- version in spec out of the agentConfig
2nd EDIT:
- Add the identifier mac to the bmh spec file
- remove OS version from the images to make it easy to be updated

### CODE REVIEW

**Comment by alknopfler** [comment]
Also tested in the ATIP examples:
https://github.com/suse-edge/atip/pull/19

**Comment by alknopfler** [comment]
> Thanks! Could it be that you missed a couple of instances?
> 
> ```
> > git remote -v
> origin	git@github.com:alknopfler/suse-edge.github.io.git (fetch)
> origin	git@github.com:alknopfler/suse-edge.github.io.git (push)
> > ack "controlplane.cluster.x-k8s.io/v1alpha1" asciidoc
> 
> asciidoc/components/rke2.adoc
> 54:apiVersion: controlplane.cluster.x-k8s.io/v1alpha1
> 
> asciidoc/product/atip-automated-provision.adoc
> 680:    apiVersion: controlplane.cluster.x-k8s.io/v1alpha1
> 713:apiVersion: controlplane.cluster.x-k8s.io/v1alpha1
> 952:    apiVersion: controlplane.cluster.x-k8s.io/v1alpha1
> 990:apiVersion: controlplane.cluster.x-k8s.io/v1alpha1
> 1407:apiVersion: controlplane.cluster.x-k8s.io/v1alpha1
> 1631:apiVersion: controlplane.cluster.x-k8s.io/v1alpha1
> 1752:apiVersion: controlplane.cluster.x-k8s.io/v1alpha1
> 
> asciidoc/product/atip-lifecycle.adoc
> 65:apiVersion: controlplane.cluster.x-k8s.io/v1alpha1
> 
> asciidoc/quickstart/metal3.adoc
> 485:    apiVersion: controlplane.cluster.x-k8s.io/v1alpha1
> 504:apiVersion: controlplane.cluster.x-k8s.io/v1alpha1
> ```

This is basically my PR

**Comment by fsundermeyer** [comment]
Just noticed it after hitting "Comment". Sorry, was on the wrong branch ;-(

**Review by fsundermeyer — APPROVED** [review]
LGTM

File: asciidoc/quickstart/metal3.adoc [hunk 1]
```diff
@@ -512,6 +512,7 @@ spec:
     kind: Metal3MachineTemplate
     name: sample-cluster-controlplane
   replicas: 1
+  version: v1.30.5+rke2r1
```
[the remark this answers is not in the document]
Comment (alknopfler) [reply asciidoc/quickstart/metal3.adoc:515]: sure!

[the remark this answers is not in the document]
Comment (alknopfler) [reply asciidoc/quickstart/metal3.adoc]: changed

**Comment by alknopfler** [comment]
@hardys I've also added the changes we commented this week:
- identifier: mac-address for the static dhcp-less environment
- remove the OS version from the EIB output image name to make it easy to be updated every release

File: asciidoc/product/atip-management-cluster.adoc [hunk 2]
```diff
@@ -974,7 +974,12 @@ This section describes how to prepare the image for air-gap environments showing
 [#mgmt-cluster-image-definition-file-airgap]
 ==== Modifications in the definition file
 
-The `mgmt-cluster.yaml` file must be modified to include the `embeddedArtifactRegistry` section with the `images` field set to all container images to be included into the EIB output image. The `images` field must contain the list of all container images to be included in the output image. The following is an example of the `mgmt-cluster.yaml` file with the `embeddedArtifactRegistry` section included:
+The `mgmt-cluster.yaml` file must be modified to include the `embeddedArtifactRegistry` section with the `images` field set to all container images to be included into the EIB output image. The `images` field must contain the list of all container images to be included in the output image.
```
Comment (fsundermeyer) [inline asciidoc/product/atip-management-cluster.adoc:977]: The two sentences include duplicated information. I suggest something like the following:

```suggestion
The `mgmt-cluster.yaml` file must be modified to include the `embeddedArtifactRegistry` section.
In this section the `images` field must contain the list of all container images to be included in the output image.
```

File: asciidoc/product/atip-management-cluster.adoc [hunk 3]
```diff
@@ -974,7 +974,12 @@ This section describes how to prepare the image for air-gap environments showing
...
+
+[NOTE]
+====
+The following is only an example of the `mgmt-cluster.yaml` file with the `embeddedArtifactRegistry` section included. Please take care of the image version depending on the components needed.
```
Comment (fsundermeyer) [inline asciidoc/product/atip-management-cluster.adoc:981]: Not sure I fully understand the second sentence. How about;

```suggestion
The following is an example of the `mgmt-cluster.yaml` file with the `embeddedArtifactRegistry` section included.
Make sure to the listed images contain the component versions you need.
```

**Review by fsundermeyer — CHANGES_REQUESTED** [review]

File: asciidoc/product/atip-management-cluster.adoc (hunk 2 above)
Comment (alknopfler) [reply asciidoc/product/atip-management-cluster.adoc:977]: makes sense! changed! Thanks

File: asciidoc/product/atip-management-cluster.adoc (hunk 3 above)
Comment (alknopfler) [reply asciidoc/product/atip-management-cluster.adoc:981]: much better! Thanks

Comment (fsundermeyer) [reply asciidoc/product/atip-management-cluster.adoc]: Actually I added a superfluous  "to". Should be: Make sure the listed images contain the component versions you need.

Sorry.

File: asciidoc/quickstart/metal3.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (hardys) [reply asciidoc/quickstart/metal3.adoc]: I don't think the attribute replacement is working - we need to add `subs="attributes"` to the yaml blocks where the attribute is referenced e.g:

```
git diff
diff --git a/asciidoc/quickstart/metal3.adoc b/asciidoc/quickstart/metal3.adoc
index c5aea00..505737e 100644
--- a/asciidoc/quickstart/metal3.adoc
+++ b/asciidoc/quickstart/metal3.adoc
@@ -466,7 +466,7 @@ To deploy the controlplane we define a yaml manifest similar to the one below, w
 Note for simplicity this example assumes a single-node controlplane, where the BareMetalHost is configured with an IP of `192.168.125.200` - for more
 advanced multi-node examples please see the <<atip-automated-provisioning, ATIP documentation>>
 
-[,yaml]
+[,yaml,subs="attributes"]
 ----
 apiVersion: cluster.x-k8s.io/v1beta1
 kind: Cluster
@@ -602,7 +602,7 @@ Similar to the controlplane we define a yaml manifest, which contains the follow
 * Metal3MachineTemplate defines the OS Image to be applied to the BareMetalHost resources, and the hostSelector defines which BareMetalHosts to consume
 * Metal3DataTemplate defines additional metaData to be passed to the BareMetalHost (note networkData is not currently supported in the Edge solution)
 
-[,yaml]
+[,yaml,subs="attributes"]
 ----
 apiVersion: cluster.x-k8s.io/v1beta1
 kind: MachineDeployment
```

[the remark this answers is not in the document]
Comment (alknopfler) [reply asciidoc/quickstart/metal3.adoc]: you're right! good catch! I've changed

**Review by hardys — APPROVED** [review]

**Review by fsundermeyer — APPROVED** [review]
Thanks for making teh changes

**Pull request merged**